source/frontend/src/features/properties/map/compensation/update/UpdateCompensationRequisitionForm.tsx

```diff
@@ -62,83 +111,156 @@ const UpdateCompensationRequisitionForm: React.FC<CompensationRequisitionFormPro
+                  {acquisitionFile.product?.code ?? ''}
+                </SectionField>
+                <SectionField label="Business function" labelWidth={'4'}>
+                  {acquisitionFile.project?.businessFunctionCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Work activity" labelWidth={'4'}>
+                  {acquisitionFile.project?.workActivityCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Cost type" labelWidth={'4'}>
+                  {acquisitionFile.project?.costTypeCode?.code ?? ''}
+                </SectionField>
+                <SectionField label="Fiscal year" labelWidth="4" contentWidth="4" required>
+                  <Select field="fiscalYear" options={fiscalYearOptions} placeholder="Select..." />
+                </SectionField>
+                <SectionField label="STOB" labelWidth="4" contentWidth="4" required>
+                  <Select field="stob" options={yearlyFinancialOptions} placeholder="Select..." />
+                </SectionField>
+                <SectionField label="Service line" labelWidth="4" required>
+                  <Select
+                    field="serviceLine"
+                    options={chartOfAccountsOptions}
+                    placeholder="Select..."
+                  />
+                </SectionField>
+                <SectionField label="Responsibility centre" labelWidth="4" required>
+                  <Select
+                    field="responsibilityCentre"
+                    options={responsiblityCentreOptions}
+                    placeholder="Select..."
+                  />
+                </SectionField>
+              </Section>
+
+              <Section header="Payment" isCollapsable initiallyExpanded>
+                <SectionField label="Payee">
+                  <Select
+                    options={dummyOptions}
+                    field={withNameSpace('payees.0', 'acquisitionOwnerId')}
+                  />
+                </SectionField>
+                <SectionField label="Payment in Trust?">
+                  <Check field={withNameSpace('payees.0', 'cheques[0].isPaymentInTrust')} />
+                </SectionField>
+                <SectionField label="GST number" tooltip="Include GST # if applicable">
+                  <Input field={withNameSpace('payees.0', 'cheques[0].gstNumber')}></Input>
+                </SectionField>
+                <SectionField label="Amount (before tax)">
+                  <FastCurrencyInput
+                    field={withNameSpace('payees.0', 'cheques[0].pretaxAmount')}
+                    formikProps={formikProps}
+                    disabled
+                  />
+                </SectionField>
+                <SectionField label="GST amount">
+                  <FastCurrencyInput
+                    field={withNameSpace('payees.0', 'cheques[0].taxAmount')}
+                    formikProps={formikProps}
+                    disabled
```

Comment:
despite this being disabled in the mockups, we need to allow these to be user-overridable. This allows the user to manually work around situations where the gst constant has been changed but they still need to use the old value.

This should follow the existing patterns, where the user overriden value may be replaced whenever the user changes any values that are required for the calculation.

Reply:
enabled